config/install/field.field.media.image.field_image_storage.yml

```diff
@@ -0,0 +1,27 @@
+uuid: 1a7e8c9e-d18f-42a6-8e7c-632c9dde5aaf
+langcode: en
+status: true
+dependencies:
```

Comment:
Sadly, no.  It's a manual process.  I add them in a text editor after exporting them.